src/main/channels/audio/GetManyAudiosChannel.ts

```
@@ -0,0 +1,73 @@
+import { GET_MANY_AUDIOS_CHANNEL } from '@shared/channels';
+import { GetManyAudiosParams } from '@shared/interfaces/Audio';
+import { instanceToPlain } from 'class-transformer';
+import { IpcMainEvent } from 'electron';
+import log from 'electron-log/main';
+import { existsSync, promises as fs } from 'fs';
+import * as path from 'path';
+
+import { IpcRequest } from '../../../shared/interfaces/IpcRequest';
+import { AppDataSource } from '../../data-source';
```

ESLint

[error] 'AppDataSource' is defined but never used.

(@typescript-eslint/no-unused-vars)